plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -64,7 +64,7 @@ type SchedulerServer struct {
 func NewSchedulerServer() *SchedulerServer {
 	s := SchedulerServer{
 		Port:              ports.SchedulerPort,
-		Address:           net.ParseIP("127.0.0.1"),
+		Address:           net.ParseIP("0.0.0.0"),
```

Comment:
What kind of data do these APIs expose?
And I assume that in the general case this would still only be accessible from within the cluster, is that right?